keyboards/dtd/ldmlKeyboard.dtd

```diff
@@ -155,8 +155,8 @@ Please see CLDR-15034 for the latest information. -->
 <!ELEMENT layer ( row+, special* ) >
 <!ATTLIST layer id NMTOKEN #IMPLIED >
     <!--@MATCH:any-->
-<!ATTLIST layer modifier NMTOKEN #IMPLIED >
-    <!--@MATCH:any-->
+<!ATTLIST layer modifier NMTOKENS #IMPLIED >
+    <!--@MATCH:regex/^(none|(\b(alt[LR]?)?\b[ ]?\b(ctrl[LR]?)?\b[ ]?\b(shift)?\b[ ]?\b(caps)?\b))$-->
```

Comment:
OK, so in the Keyman repo, you sorted the tokens alphabetically in the string before processing. Should we consider doing that here too? (`"alt[LR] caps ctrl[LR] shift"` -- case against: it feels awkward throwing caps in the middle)

Reply:
didn't we talk about this being potentially open ended set for cases with extra shift states?

Reply:
i'll make it open ended

Reply:
done